client/src/components/product/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { useGetProductList, useAddCartMutation } from '@/api';
+import { Product } from '@/dto';
+import { Link } from 'react-router-dom';
+
+const ProductList = () => {
+  const { data, error, isLoading } = useGetProductList();
+  const { mutate: onAddCart } = useAddCartMutation();
+  const onClickImg = (product: Product) => {
+    onAddCart(product);
+  };
+  return isLoading ? (
+    <div>Loading...</div>
+  ) : (
+    <section className="product-container">
+      {data?.map((product) => (
+        <div className="product-item" key={product.id}>
+          <img src={product.imageUrl} alt={product.name} />
+          <div className="flex justify-between p-5">
+            <div className="product-info">
+              <span className="product-info__name">{product.name}</span>
+              <span className="product-info__price">{product.price}원</span>
+            </div>
+            <Link to="/cart">
+              <div onClick={() => onClickImg(product)}>
```

Comment:
필요한 추가 동작이 없다면, 바로 onAddCart() 해주면 좋을 것 같습니다.

추가 동작 있을 때, 이벤트 핸들러의 네이밍을 onClickImg가 좋은 것인가에 대한 의문이 있습니다.
`onClick={onClickImg}` 등의 발생시키는 동작을 담아내는 네이밍보다,
`onClick ={addToCart}` 등의 Click시 발생하는 동작을 의미적으로 담는 게 나은 방법이라고 생각하는데, 주로 사용되는 어떤 네이밍컨벤션 기준이 있는지 궁금하네요.
```suggestion
              <div onClick={() => onAddCart(product)}>
```